crates/solidity/inputs/language/src/definition.rs

```diff
@@ -3794,7 +3794,7 @@ codegen_language_macros::compile!(Language(
                             definitions = [
                                 // Allows unicode characters:
                                 TokenDefinition(
-                                    enabled = Range(from = "0.4.12", till = "0.7.0"),
+                                    enabled = Till("0.7.0"),
```

Comment:
i see the two tab "\t" tests are also affected. Is it included in these changes?